src/main/java/com/box/sdk/BoxFile.java

```diff
@@ -94,6 +97,40 @@ public BoxSharedLink createSharedLink(BoxSharedLink.Access access, Date unshareD
     }
 
     /**
+     * Adds a new task to this file. The task can have an optional message to include, and a due date
+     * @param action the action the task assignee will be prompted to do
+     * @param message an optional message to include with the task
+     * @param dueAt the day at which this task is due
+     * @return information about the newly added task
+     */
+    public BoxTask.Info addTask(String action, String message, Date dueAt) {
+        JsonObject itemJSON = new JsonObject();
+        itemJSON.add("type", "file");
+        itemJSON.add("id", this.getID());
+
+        JsonObject requestJSON = new JsonObject();
+        requestJSON.add("item", itemJSON);
+        requestJSON.add("action", action);
+
+        if (message != null) {
```

Comment:
The message can be non-null but empty. So adding an isEmpty check will be a good idea.